### PR TITLE
Renamed analytics events

### PR DESCRIPTION
**Pull request by amykhailiutenko-sc**
Renamed analytics events

### CODE REVIEW

**Comment by amykhailiutenko-sc** [comment]
Renamed events according its analogs on iOS side.
Ticket: https://applicaster.atlassian.net/browse/MS2-15

**Comment by falloutfox** [comment]
iOS PR aligning event names: https://github.com/applicaster/JWPlayer-Plugin-iOS/pull/16

**Comment by YoniOsteen** [comment]
Please make sure the event names are aligned with iOS (or vice-versa). They have Start Video and Play Video and you have Start VOD and Play VOD. This is going to require the customer to add two events together and all sorts off manual workarounds in the console if they are not precisely aligned.

**Comment by amykhailiutenko-sc** [comment]
Updated values.

File: src/main/java/com/applicaster/jwplayerplugin/JWPlayerActivity.java [hunk 1]
```diff
@@ -31,6 +31,7 @@
 
     private static final String PLAYABLE_KEY = "playable_key";
     private static final String PERCENTAGE_KEY = "percentage";
+    private static final String ADVERTISEMENT_POSITION_KEY = "advertisement_position_key";
```
Comment (guysarfatty) [inline src/main/java/com/applicaster/jwplayerplugin/JWPlayerActivity.java]: Please remove "_key" from the value of the String.

**Comment by amykhailiutenko-sc** [comment]
updated value

File: src/main/java/com/applicaster/jwplayerplugin/JWPlayerActivity.java [hunk 2]
```diff
@@ -168,7 +169,7 @@ public void onTime(TimeEvent timeEvent) {
 
         Map<String, String> params = new HashMap<>(analyticsParams);
         params.put(PERCENTAGE_KEY, String.valueOf(trackedPercentage));
-        AnalyticsAgentUtil.logEvent("VOD Item: Percentage watched", params);
+        AnalyticsAgentUtil.logEvent("Watch VOD percentage", params);
```
Comment (guysarfatty) [inline src/main/java/com/applicaster/jwplayerplugin/JWPlayerActivity.java]: One last request - capital "P" in "percentage". Thanks

**Pull request merged**